index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
Your `head` is missing a [proper `title` element](https://mica-web.github.io/learn/html/best-practices#include-a-proper-title-in-your-html-head). This is to add document information to browser windows and tabs and when a user bookmarks a site, among other things. It can include no other HTML or code, but it can include spaces, words or numbers.